la.html

```diff
@@ -0,0 +1,143 @@
+<!doctype html>
+<html>
+<head>
+<title>Our Funky HTML Page</title>
+<link rel="stylesheet" type="text/css" href="./vlog.css"/>
+<meta name="description" content="Our first page">
+<meta name="keywords" content="html tutorial template">
+</head>
+<body>
+
+<div class="main-menu">
+    <div class="main-menu-top__logo"> 
+            <img class="main-menu__logo-big" src="linklogo.png">
+        <div class="main-menu__headline"> <h1>Jack</h1> </div>
+        <div class="main-menu__headline-small"> Web Developer </div>
+    </div>
+    <div class="vertical-menu">
+        <a href="#" class="main-menu__about">About</a>
+        <a href="#" class="main-menu__my-skills">My Skills</a>
+        <a href="#" class="main-menu__work">Work</a>
+        <a href="#" class="main-menu__contact">Contact</a>
+        <a href="#" class="main-menu__blog">Blog</a>
+    </div> 
+    <div class="media">
+        <a href="https://www.linkedin.com/in/dmitrypm/"> <img class="main-menu__linkedin" src="linklogo.png"></a>
+        <a href="https://github.com/DmitriiTsy"> <img class="main-menu__git" src="Octocat.png"> </a>
+
+    </div>
+</div>
+</div>
+
+<div class="first-page">
+  <div class="first-page__header">
+    <span>H</span>
```

Comment:
let's do this another way, I'd use span tag for a word, not for a letter, smth like
<span>Word</span>
<br />
<span>Another Word</span>

Reply:
Done. Will do pull request tomorrow